processor/otel/consumer.go

```diff
@@ -517,12 +520,21 @@ func translateSpan(span pdata.Span, metadata model.Metadata, event *model.Span)
 					netPeerName = stringval
 				}
 
-			// messaging
-			//
-			// TODO(axw) translate OpenTelemtry messaging conventions.
-			case "message_bus.destination":
+			// messaging.*
+			case "message_bus.destination", conventions.AttributeMessagingDestination:
 				message.QueueName = stringval
 				isMessagingSpan = true
+			case conventions.AttributeMessagingOperation:
+				if stringval == "" && span.Kind() == pdata.SpanKindPRODUCER {
```

Comment:
Won't the attribute just be omitted altogether, rather than being present with an empty string?

Reply:
ah, yeah. will do the check later in the code.